<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDisplayPreview</class>
 <widget class="QWidget" name="MainDisplayPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
font-family:&quot;微软雅黑&quot;;
background-color: transparent;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="whiteBoardPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="leftBarContainer" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>32</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QWidget" name="toolBarLeft" native="true">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>360</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>360</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget#toolBarLeft{
background:#FFFFFF;
border-radius: 5px;
}
QPushButton {
font-size:18px;
color:#555555;
font-family:&quot;lviconfont&quot;;
}

QPushButton:checked {
background: #F3F3F3;
border-radius: 4px;
}

QPushButton:hover {
background: #F3F3F3;
border-radius: 4px;
}

QLabel#line, QLabel#line_2{
border: 1px solid #F1F1F1;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>7</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>7</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="selectBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>选择</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="multiSelectBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>框</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="penBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>画笔</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="textBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>文本</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="laserBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>激光</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="line_2">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>1</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>1</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="picBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>图片</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="uploadPptBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>ppt</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="line">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>1</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>1</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="eraseBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="text">
                <string>橡皮</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearBtn">
               <property name="minimumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #BDBDBD;</string>
               </property>
               <property name="text">
                <string>清空</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottomBarContainer" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <widget class="QWidget" name="toolbarReUndo" native="true">
          <property name="geometry">
           <rect>
            <x>32</x>
            <y>30</y>
            <width>90</width>
            <height>36</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#toolbarReUndo{
background:#FFFFFF;
border-radius: 5px;
}
QPushButton {
font-size:18px;
color:#555555;
font-family:&quot;lviconfont&quot;;
}

QPushButton:checked {
background: #F3F3F3;
border-radius: 4px;
}

QPushButton:hover {
background: #F3F3F3;
border-radius: 4px;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="unBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>撤销</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>重做</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="toolBarScale" native="true">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>130</width>
            <height>36</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#toolBarScale{
background:#FFFFFF;
border-radius: 5px;
}
QPushButton {
font-size:20px;
color:#555555;
font-family:&quot;lviconfont&quot;;
}

QPushButton:checked {
background: #F3F3F3;
border-radius: 4px;
}

QPushButton:hover {
background: #F3F3F3;
border-radius: 4px;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="zoomInBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>缩小</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="scaleLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 14px;
color: #333333;
border: 1px solid #DADADA;
font-family:&quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>100%</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="zoomOutBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>放大</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="toolBarBottom" native="true">
          <property name="geometry">
           <rect>
            <x>305</x>
            <y>30</y>
            <width>220</width>
            <height>36</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>36</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#toolBarBottom{
background:#FFFFFF;
border-radius: 5px;
}

QPushButton {
font-size:16px;
color:#555555;
font-family:&quot;lviconfont&quot;;
}

QPushButton:checked {
background: #F3F3F3;
border-radius: 4px;
}

QPushButton:hover {
background: #F3F3F3;
border-radius: 4px;
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="firstBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>最前</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="preBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>前一页</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pageLabel">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font-size: 14px;
color: #333333;
border: 1px solid #DADADA;
font-family:&quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>1/1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>后一页</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="lastBtn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>最后</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newPage">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>新的</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
